04.DOM_manipulation/src/domManipulation.js

```diff
@@ -1,3 +1,26 @@
-var dom = function(){
-  // code goes here
+var dom = {
+  remove: function(selectedElement) {
+    selectedElement.remove();
+  },
+  append: function(selectedElement, newElement) {
+    selectedElement.appendChild(newElement);
+  },
+  prepend: function(selectedElement, newElement) {
+    selectedElement.prepend(newElement);
+  },
+  after: function(selectedElement, newElement) {
+    selectedElement.insertAdjacentElement('afterend', newElement);
+  },
+  before: function(selectedElement, newElement) {
+    selectedElement.insertAdjacentElement('beforebegin', newElement);
+  },
+  val: function(selectedElement) {
+    if(selectedElement.tagName === 'SELECT') {
```

Comment:
```suggestion
    if (selectedElement.tagName === 'SELECT') {
```
Add spaces after if statements